.github/workflows/tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           - ":presto-tests -P ci-only-distributed-queries"
           - ":presto-tests -P ci-only-aggregation-queries"
           - ":presto-tests -P ci-only-plan-determinism"
-          - ":presto-tests -P ci-only-resource-manager" 
+          - ":presto-tests -P ci-only-resource-manager"
```

Comment:
Change is not related?

Reply:
@pratyakshsharma Can you please remove this change? It seems unrelated.